entity-service/src/main/kotlin/com/egm/stellio/entity/model/Attribute.kt

```diff
@@ -60,6 +62,11 @@ open class Attribute(
             )
         }
 
+        datasetId?.run {
+            if (this != "default")
```

Comment:
"default" should be a constant, so if this is changing the modification is straightforward